newsletter-january-2023.rst

```diff
@@ -0,0 +1,83 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023! In this newsletter, we present the updates happening in 2022's last month and since the :doc:`previous newsletter </newsletter-december-2022>`.
+
+.. Commenting out this stuff, isn't it better that 
+.. 
+.. News and updates
+.. ----------------
+.. 
+.. Here are the latest updates from our team:
+
+- 📹️ Eric's talk at DjangoCon US can be watched here: `Documenting Django Code in 2022`_
+- 🛠️ We are busy reorganizing our own documentation. Since last newsletter, we have opened or merged 20 documentation PRs. We look forward to a "show and tell" about how all of this was refactored using the `Diátaxis Framework`_.
+- 🚢️ Sphinx 6.0 and 6.1 have been released. Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- ⚙️ Add new stuff here
+- 💡️ Choose your own emoji
+- 🔍️ 404 pages are being improved by contextualization the user message, giving relevant guidance to readers and project owners.
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _Documenting Django Code in 2022: https://www.youtube.com/watch?v=mqn0D4xat58
+.. _Diátaxis Framework: https://diataxis.fr/
+
+Upcoming features
+-----------------
+
+- We're working on improving our integration with `Material for MkDocs <https://squidfunk.github.io/mkdocs-material/>`_, which is a great theme for `MkDocs <https://www.mkdocs.org/>`_ documentation projects.
+- Many improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- A search redesign to make it nicer across our dashboard and in-doc search experiences. 
+- 
+
+Possible issues
+---------------
+
+Sphinx 6 and 6.1 were released very recently and you may experience issues if you upgrade or if you have not pinned your documentation dependencies.
```

Comment:
```suggestion
Sphinx 6.0 and 6.1 were released very recently and you may experience issues if you upgrade or if you have not pinned your documentation dependencies.
```

I think we should tell the users what to do here or how to fix the issue or something. Some guidance in case they hit an issue.

I don't think we have to explain _the solutions_ here, but at least a link somewhere to read more --or how to pin the dependencies, etc.